conda/multisheller/matlab-whole-body-simulator_activate.msh

```diff
@@ -0,0 +1,21 @@
+if_(is_set("COMSPEC")).then_([
+  sys.list_append("YARP_DATA_DIRS", path.join(env("CONDA_PREFIX"), "Library\\share\\RRbot")),
+  sys.list_append("ROS_PACKAGE_PATH", path.join(env("CONDA_PREFIX"), "Library\\share")),
+  sys.list_append("AMENT_PREFIX_PATH", path.join(env("CONDA_PREFIX"), "Library\\share")),
+  sys.list_append("GAZEBO_MODEL_PATH", path.join(env("CONDA_PREFIX"), "Library\\share\\RRbot\\robots")),
+  sys.list_append("MATLABPATH", path.join(env("CONDA_PREFIX"), "Library\\mex\\+mwbs\\RobotDynamicsWithContacts"))
```

Comment:
I guess this can be removed?

Reply:
Ah yes, a leftover. Done.

Reply:
Removed other leftovers in the documentation. Refer to https://github.com/robotology/robotology-superbuild/pull/689/commits/b42bf48c0b8b6a527e4ca998e1780f3f40d70b2a.